app/components/op_primer/flash_component.rb

```diff
@@ -45,5 +45,17 @@ def initialize(**system_arguments)
 
       super
     end
+
+    def render_as_turbo_stream(...)
+      return unless render?
+
+      super
+    end
```

Comment:
I have only changed this behaviour in `FlashComponent` as there may be a use case for rendering empty `<turbo-stream/>` tags for other Turbo actions that I haven't considered.